src/app/utils.ts

```diff
@@ -346,3 +346,12 @@ export function detectBrowserLanguage(defaultLanguage: string = "en"): string {
   // Fallback to a default language if none is detected
   return defaultLanguage;
 }
+
+// If available, adds currently selected pair to the page title.
+export function updatePageTitle(pairName: string) {
+  if (pairName === "") {
+    document.title = "DeXter";
+  } else {
+    document.title = `DeXter • ${pairName}`;
+  }
+}
```

Comment:
Will this logic be used only once in the trading page? If yes, it's better to keep it there, not to grow the utils indefinitely. If it's for all pages then maybe editing the head->title tag in `src/app/layout.tsx`? 

If it will be useful for several pages but not for all, the name of the function should be something different not `updatePageTitle`,  maybe `addPairToTitle`. Or if improving reusability a bit more - create something like a custom hook instead of a util function, so that that pairSelector would also go there and if a page needs the pair name in the title it would just call `useTitlePageWithPair()` or something like this.

Reply:
Good questions. I agree the util seems unneccessary, removed it and kept the logic directly inside the trade component.

Concerning using the pair in the title throughout the webapp: I don't think that is planned. The pair should only be shown in the "Trade" page. While I do like the idea of creating a custom hook, I think a pragmatic solve here is sufficient :)

Let me know what you think!

Reply:
just in /trade page is perfect 👍